server/Services/Internal/BookingService.cs

```diff
@@ -42,6 +45,23 @@ public async Task<BookingDto> CreateBookingAsync(CreateBookingRequest bookingReq
             }
         }
 
+        public async Task CreateReaquringBookingAsync(IEnumerable<SeatAllocationDetails> seatAssignmentDetails, DateTime bookingDateTime)
```

Comment:
Typo for "RecurringBooking"? og det bør også har sammen navn på interface.

Reply:
Fikset